core/src/test/java/org/apache/calcite/test/TableInRootSchemaTest.java

```diff
@@ -75,14 +74,14 @@
     final ResultSetMetaData resultSetMetaData = resultSet.getMetaData();
     assertThat(resultSetMetaData.getColumnName(1), equalTo("A"));
     assertThat(resultSetMetaData.getTableName(1), equalTo("SAMPLE"));
-    assertThat(resultSetMetaData.getSchemaName(1), nullValue());
+    assertThat(resultSetMetaData.getSchemaName(1), equalTo(""));
     assertThat(resultSetMetaData.getColumnClassName(1),
         equalTo("java.lang.String"));
     // Per JDBC, column name should be null. But DBUnit requires every column
     // to have a name, so the driver uses the label.
```

Comment:
update doc ?

Reply:
The doc is for column name, which is not changed by this PR.